incubator/hnc/hack/ci-test.sh

```diff
@@ -32,8 +32,11 @@ wget https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.0.0-alp
 tar -zxvf kubebuilder_2.0.0-alpha.1_linux_amd64.tar.gz
 mv kubebuilder_2.0.0-alpha.1_linux_amd64 /usr/local/kubebuilder
 
+echo "Installing 'controller-gen'"
+GO111MODULE=on go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.2.5
+
```

Comment:
Why did we not need this before, but we need it now? Shouldn't the fact that it's in `go.mod` be enough?

The fewer steps and versions we have in this file, the better IMO.

Reply:
Looks like if I do not explicitly get it here, when we run pull-hnc-test, it will get v0.2.1 version of sigs.k8s.io/controller-tools/cmd/controller-gen. Even though we set it to v0.2.5 in go.mod. I am not sure the reason. Looks like when running pull-hnc-test does not get all packages listed in go.mod (e.g., it doesn't get controller-runtime from the log below):

https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_multi-tenancy/504/pull-hnc-test/1237032233525579778/build-log.txt

Reply:
(I looked at the log and noticed that it was the first line after calling `make`, so I checked the makefile)

Reply:
Ah! That's the problem! Thanks a lot for pointing it out!